src/common/helpers/ImportHelpers.ts

```diff
@@ -1,5 +1,7 @@
 import { Position, TextEditor } from 'vscode';
 
+const REGEX_IGNORED_LINE = /^\s*(?:\/\/|\/\*\*|\*\/|\*|(['"])use strict\1)/
```

Comment:
- Ensuring quote match the usual way (group backreference)
- Switching to non-capturing group for better perf (as we won't be needing the group capture)
- Stripping useless costly `g` flag